{\rtf1\ansi\ansicpg1252\cocoartf2822
\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
{\*\expandedcolortbl;;}
\paperw11900\paperh16840\margl1440\margr1440\vieww29200\viewh17820\viewkind0
\pard\tx720\tx1440\tx2160\tx2880\tx3600\tx4320\tx5040\tx5760\tx6480\tx7200\tx7920\tx8640\pardirnatural\partightenfactor0

\f0\fs24 \cf0 --- Pods-PharmAI-resources.sh.bak\
+++ Pods-PharmAI-resources.sh\
@@ -55,7 +55,7 @@\
 rsync -avr --copy-links --no-relative --exclude '*/.svn/*' --files-from="$RESOURCES_TO_COPY" / "$TMP_RES_PATH"\
\
 cd "$TMP_RES_PATH"\
-find . -type f  | while read -r file; do\
+find . -type f ! -name "Localizable.strings" | while read -r file; do\
   full_dest="$\{TARGET_BUILD_DIR\}/$\{UNLOCALIZED_RESOURCES_FOLDER_PATH\}/$file"\
   mkdir -p "$(dirname "$full_dest")"\
   cp "$file" "$full_dest"}